CHANGELOG.md

```diff
@@ -5,6 +5,15 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 
+## Unreleased as of Sprint 87 ending 2018-06-04
+
+### Added
+- Adding Physical Switches support [(#370)](https://github.com/ManageIQ/manageiq-api/pull/370)
```

Comment:
can you update to reflect https://github.com/ManageIQ/manageiq-api/pull/389 ?  Thanks.